### PR TITLE
Feature/semantic html

### PR DESCRIPTION
**Pull request by knpausch**
Summary of the work done:

- Reorganizes HTML elements to be semantic on index.html and scripts.js files according to the HTML elements reference [guide](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)

This branch includes:

- renames all div to an alternative element (article, section, ul, menu, etc.)
- minor CSS adjustments to remove default element padding/margins 
- renames elements into partent/child style HTML according to the HTML elements reference

Contributed to code:

- Kiko

Needs to review:

- Jen
- Lauren

Additional notes about any issues/questions remaining around this code:

- Let me know any thoughts. I did my best to maintain the original look while also incorporating the rules in the HTML elements reference
- This branch did not seem to impact our Lighthouse score however semantic HTML is required in the project's rubric

Indicate anything that still needs to be addressed

- This is prior to Lauren's Pantry list fix (when you click pantry button and its blank under 'ingredients' and 'amount'). We could merge this branch then when we merge her newer pantry version, I can adjust any remaining HTML elements then

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -126,28 +124,29 @@ <h2>Total Cost</h2>
     </section>
     <section class="pantry hidden" id="pantry-view">
       <section class="pantryContents">
-        <div class="dropDropMenu">
+        <menu class="dropDropMenu">
           <label for="ingredients">Add Ingredient to Pantry: </label>
             <select name="ingredients" id="ingredient-drop-down-menu" class="selectMenu">
             </select>
             <form class="quantity">
```
Comment (FrazierLE) [inline dist/index.html:131]: I woke up thinking that I should've put this in a form so that's awesome!

File: dist/index.html [hunk 2]
```diff
@@ -126,28 +124,29 @@ <h2>Total Cost</h2>
...
             </form>
             <button type="button" class="addButton" id="add-button">Add to Pantry</button>
-        </div>
-        <div class="pantryItems">
-            <div class="ingredientsPantry">
-            <div class="pantryHeading">
+        </menu>
+        <article class="pantryItems">
+          <article class="ingredientsPantry">
+            <header class="pantryHeading">
```
Comment (FrazierLE) [inline dist/index.html:138]: Would this be a header?

File: dist/index.html [hunk 3]
```diff
@@ -126,28 +124,29 @@ <h2>Total Cost</h2>
...
               <h3>Ingredients</h3>
               <h3>Amount</h3>
-            </div>
-            <div class="table" id="pantry-table">
-              <div class="boxI"></div>
-              <div class="boxA"></div>
+            </header>
```
Comment (FrazierLE) [inline dist/index.html:141]: Thank you for doing this. I did a pretty sloppy job of throwing divs everywhere and planned to go back and clean that up. I appreciate you

File: dist/index.html [hunk 4]
```diff
@@ -11,109 +11,107 @@ <h1>What's Cookin'?</h1>
...
-      <section class="filter-section" id="filterSection">
-        <form class="search">
+      <form class="filter-section" id="filterSection">
+        <fieldset class="search">
             <input type="text" placeholder="Search..." id="search-bar">
-          <button type="button" class="submit-search-button" id="submit-search-button">Search</button>
-        </form>
-        <form class = "tags" id="tags">
+            <button type="button" class="submit-search-button" id="submit-search-button">Search</button>
+        </fieldset>
```
Comment (FrazierLE) [inline dist/index.html:27]: I've never seen fields or datalist before so that's cool

**Review by FrazierLE — APPROVED** [review]

File: dist/index.html (hunk 2 above)
Comment (jmyacobn) [reply dist/index.html:138]: I think a header is reserved for a header on the page. The <header> element represents a container for introductory content or a set of navigational links. A <header> element typically contains: one or more heading elements (<h1> - <h6>), logo/icon, or authorship information

**Pull request closed without merging**